phpBB/phpbb/console/command/reparser/reparse.php

```diff
@@ -163,29 +171,39 @@ protected function execute(InputInterface $input, OutputInterface $output)
 		$this->input = $input;
 		$this->output = $output;
 		$this->io = new SymfonyStyle($input, $output);
-		$this->load_resume_data();
 
-		$name = $input->getArgument('reparser-name');
-		if (isset($name))
+		if (!$this->reparse_lock->acquire())
```

Comment:
Argh, thanks. That's the way I wanted to write it first, but then I changed it (mainly to be consistent with everything else that acquires a lock) and forgot to change the conditional too. :/